### PR TITLE
Add Haskell support to amethyst-trusty and sugilite-trusty stacks

### LINKED ISSUE #366: Support Haskell/GHC on Trusty

### PR DESCRIPTION
**Pull request by meatballhat**
Closes #366

- [x] remove `.test-branch`

### CODE REVIEW

File: ci-amethyst.yml [hunk 1]
```diff
@@ -6,7 +6,8 @@ variables:
   gce_account_file: "{{ env `GCE_ACCOUNT_FILE` }}"
   gce_project_id: "{{ env `GCE_PROJECT_ID` }}"
   image_name: travis-ci-amethyst-trusty-{{ timestamp }}
-  travis_cookbooks_branch: "{{ env `TRAVIS_COOKBOOKS_BRANCH` }}"
+  # travis_cookbooks_branch: "{{ env `TRAVIS_COOKBOOKS_BRANCH` }}"
+  travis_cookbooks_branch: meat-trusty-haskell
```
Comment (emdantrim) [inline ci-amethyst.yml]: is this supposed to still be here?

File: ci-sugilite.yml [hunk 2]
```diff
@@ -4,7 +4,8 @@ variables:
   gce_account_file: "{{ env `GCE_ACCOUNT_FILE` }}"
   gce_project_id: "{{ env `GCE_PROJECT_ID` }}"
   image_name: travis-ci-sugilite-trusty-{{ timestamp }}
-  travis_cookbooks_branch: "{{ env `TRAVIS_COOKBOOKS_BRANCH` }}"
+  # travis_cookbooks_branch: "{{ env `TRAVIS_COOKBOOKS_BRANCH` }}"
+  travis_cookbooks_branch: meat-trusty-haskell
```
Comment (emdantrim) [inline ci-sugilite.yml]: this, as well

File: ci-amethyst.yml (hunk 1 above)
Comment (meatballhat) [inline ci-amethyst.yml]: ack, meant to have a todo in the description to remove that

**Comment by meatballhat** [comment]
@emdantrim that push build looks like a legit failure since I pushed without first regenerating the system-info YAML inputs 😅

**Pull request merged**